README.md

```diff
@@ -102,9 +110,18 @@ This list was inspired by the [awesome-broadcasting list](https://github.com/ebu
 - [Awesome Pipeline](https://github.com/pditommaso/awesome-pipeline) – a curated list of pipeline toolkits.
 - [open-workflows](https://github.com/amiaopensource/open-workflows) – a curated list of resources and workflows for a/v archivists.
 
+## Manuals
+- [Film-Tech Equipment Manual Warehouse](http://www.film-tech.com/warehouse/index.php?category=2) - Manuals for Cinema equipment
+- [kinotehnik.net](https://kinotehnik.net/) - (Ukranian) Various equipment manuals in English
+
+## Media Block (Server) Serial Numbers and Certificates
+- [Server Serial Guide](https://picnickdm.freshdesk.com/support/solutions/articles/17000069284-server-serial-guide) - Serial number format for most servers
+- [How to identify your Servers Serial and Media Block Number](https://picnickdm.freshdesk.com/support/solutions/articles/17000069283-how-to-identify-your-servers-serial-and-media-block-number) - Finding the serial number on most servers
+
 ## News
 
 - [Celluloid Junkie](https://celluloidjunkie.com/) – news resource dedicated to the global film and cinema business.
+- [Cinema Tech Today](https://www.cinematech.today/) - Online edition of the Cinema Technology Magazine
```

Comment:
Their server cert has expired 05/2023, breaks the flow somewhat?

Reply:
Sure, I can remove it. Still lots of useful historical articles going back a few years. Also I think IMIS who is responsible is going to have some kind of reboot. Hopefully they will renew the cert soon.

Reply:
For now, I suggest keeping the link but using `http` instead of `https` (there will still be a warning but not a blocking screen). If you do a quick Google search, you will notice that Google links to `http` and not `https` anymore. We could also add a "(http only)" note, something like this:

```markdown
Cinema Tech Today](http://www.cinematech.today/) (http only) - online edition of the Cinema Technology Magazine
```

Even though the page seems stale, I emailed Peter & Bob from cinematech.today to fix the issue. Maybe they'll still fix the issue before the "some kind of reboot." We could then switch back to `https`.